packages/cli/src/h2.cts

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+
+const path = require('path');
+const {spawnSync} = require('child_process');
+
+let cliPath: string;
+
+try {
```

Comment:
Really tried to keep it as simple as possible. Since we already have `shopify` installed locally in CWD, this is basically just trying to find it and delegate everything.

But yeah, I guess we can call Oclif directly from here and plug cli-hydrogen, not sure what are the trade-offs.
In any case, that sounds like more complex so I will leave this ticket open until we release the stable version and have more time to test things 👍